### DISCUSSION TITLE
Fontbakery web: underlying version? update frequency?

### DISCUSSION THREAD

**Discussion by tphinney** [fonttools/fontbakery#4366] [Ideas]
@simoncozens 
I love love love the web version of Fontbakery.
But I would sure like to know which version of Fontbakery underlying code it is running. It doesn’t seem to show me any version info in the interface or report.

**Comment by simoncozens** [comment]
It's always the most recent. The first thing the Python runtime does is to literally run "pip install fontbakery". Saves us messing around updating it...

**Comment by tphinney** [comment]
That’s great! Should be made clear on the page though.

**Comment by davelab6** [comment]
Fixed in #4424
